extensions/vscode-api-tests/src/singlefolder-tests/window.test.ts

```diff
@@ -355,6 +355,14 @@ suite('window namespace tests', () => {
 		]);
 	});
 
+	test('showInputBox multiline - default value on Enter', function () {
+		const p = window.showInputBox({ value: 'foo\nbar', multiline: true });
+		return Promise.all<any>([
+			p.then(value => assert.equal(value, 'foo\nbar')),
+			commands.executeCommand('workbench.action.acceptSelectedQuickOpenItem'),
+		]);
+	});
```

Comment:
Afaict we don't have a testing strategy that would allow me to test this in user perspective way, but this is good enought ig